common/check_vm_settings.yml

```diff
@@ -16,6 +16,10 @@
 - name: "Get VM's network adapters info"
   include_tasks: vm_get_network_facts.yml
 
+- name: "Set the fact for checking VM settings"
+  ansible.builtin.set_fact:
+    vm_settings_checked: true
```

Comment:
It's better be defined after the assert task

Reply:
Ok. Will also add one task to set  value for exit_testing_when_fail. If exit_testing_when_fail is set to true, will exit ansible and not execute following test cases.